build.gradle

```diff
@@ -104,6 +104,8 @@ configure(javaProjects) {
 
   tasks.withType(JavaCompile) {
     options.compilerArgs << '-Xlint:unchecked' << '-Xlint:deprecation'
+    // This flag is only available in JDK 9+
+    options.compilerArgs.addAll(['--release', '8'])
```

Comment:
This flag ensures the generated bytecode is compatible with Java 8. (`targetCompatibility = 1.8` is not sufficient)

Reply:
Could we make this a source code comment?

Reply:
Added